modules/vuetify/index.js

```diff
@@ -0,0 +1,16 @@
+const path = require('path')
+
+module.exports = function nuxtVuetify (options) {
+  // Add assets
+  this.options.head.link.push({
+    rel: 'stylesheet',
+    type: 'text/css',
+    href: '//fonts.googleapis.com/css?family=Material+Icons'
```

Comment:
That's a good idea indeed using regex to search headers. But I think that option would be required as some users may already include a custom alternative font in their global sass files.